cypress.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = defineConfig({
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
           return {
             username: faker.name.firstName() + `${randomNumber}`,
-            email: 'test' + `${randomNumber}` + '@mail.com',
+            email: 'test' + `${randomNumber}` + '@qa.team',
             password: '12345Qwert!'
```

Comment:
The password is hardcoded. It's recommended to generate random secure passwords using a library like faker.